package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
**TODO FOR LOST POINTS**

Per lab instructions, all your work should be in a directory called ``lab-kayla``.